services/movie_session.py

```diff
@@ -0,0 +1,60 @@
+from db.models import MovieSession, Movie, CinemaHall
+from datetime import datetime
+from typing import Optional, List
+
+
+def create_movie_session(
+    movie_show_time: datetime,
+    movie_id: int,
+    cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall
+    )
+
+
+def get_movie_sessions(
+        session_date: Optional[str] = None) -> List[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
```

Comment:
This line uses `.filter()` to get movie sessions by date, which is correct in this context because you may have multiple sessions on the same date. No changes needed here.